api/src/models/nrced/administrativePenalty-nrced.js

```diff
@@ -41,7 +41,14 @@ module.exports = require('../../utils/model-schema-generator')(
 
     sourceDateAdded: { type: Date, default: null },
     sourceDateUpdated: { type: Date, default: null },
-    sourceSystemRef: { type: String, default: 'nrpti' }
+    sourceSystemRef: { type: String, default: 'nrpti' },
+
+    indexes__: [
+      {
+        fields: { summary: 'text', location: 'text', issuedTo: 'text', recordName: 'text' },
+        options: { weights: { issuedTo: 4, recordName: 3, location: 2, summary: 1 } }
+      }
+    ]
```

Comment:
Remove weights from the options property as we have no clear reason/directive to weight these fields differently at this time.  It would be nice to give it a unique name:  property however.

Same goes for all the same changes below.